.github/workflows/windows-bundle-update.yml

```diff
@@ -0,0 +1,25 @@
+name: Windows Bundle Update
+
+on:
+  workflow_dispatch
+
+jobs:
+  bundle-update:
+    runs-on: windows-latest
+    steps:
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2 # Use same Ruby version with fluent-package bundled
+      - uses: actions/checkout@v4
+      - name: Bundle Update
+        run: |
+          cd fluent-package
+          gem install bundler:2.3.27
+          bundle update -j 4
+      - uses: peter-evans/create-pull-request@v5
```

Comment:
Are there alternative way to do without Market place action.
(e.g. gh command)

https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

secrets.GITHUB_TOKEN is allowed to create PR.

Reply:
I fixed that it does not use Market place action.